Cargo.toml

```diff
@@ -13,7 +13,10 @@ serde_json = "^1.0"
 osmpbfreader = "^0.14"
 geo-types = "^0.6"
 geo = "^0.14"
-gdal = { version = "^0.7.2" }
+gdal = { version = "^0.7.2", optional=true }
 geojson = { version = "^0.19", features = ["geo-types"] }
 smartstring = "^0.2.3"
 clap = "^2.33"
+
+[features]
+default = ["gdal"]
```

Comment:
wouldn't it be better to have this not be part of the nominal build path? as in, if you want gdal, you have to explicitly ask for it?

Reply:
The primary purpose of this package is the cli, and gdal is the standard for reading/writing spatial data so I've assumed a cli user would use it.

This provides an opt-out for those who just want to use the logic of the library. Things would probably be better split into `sulu-lib` and `sulu-cli`, but that's a job for another day.

Reply:
could we raise an issue for splitting the crate into core and cli respectively? that way downstream things can pin directly to the core crate rather than having to manually adjust their builds to opt out of all default features?